components/homme/src/theta-l/share/model_init_mod.F90

```diff
@@ -121,35 +121,28 @@ subroutine model_init2(elem,hybrid,deriv,hvcoord,tl,nets,nete )
     nlev_tom=0
     if (hybrid%masterthread) write(iulog,*) "sponge layer nu_top viscosity scaling factor"
     do k=1,nlev
-       !press = (hvcoord%hyam(k)+hvcoord%hybm(k))*hvcoord%ps0
-       !ptop  = hvcoord%hyai(1)*hvcoord%ps0
-       ! sponge layer starts at p=4*ptop 
-       ! 
-       ! some test cases have ptop=200mb
-       if (hvcoord%etai(1)==0) then
-          ! pure sigma coordinates could have etai(1)=0
-          ptop_over_press = hvcoord%etam(1) / hvcoord%etam(k)  
+       if (tom_sponge_start==0) then
+          ! some test cases have ptop=200mb
+          if (hvcoord%etai(1)==0) then
+             ! pure sigma coordinates could have etai(1)=0
+             ptop_over_press = hvcoord%etam(1) / hvcoord%etam(k)  
+          else
+             ptop_over_press = hvcoord%etai(1) / hvcoord%etam(k)  
+          endif
+          ! active for p<10*ptop (following cd_core.F90 in CAM-FV)
+          ! CAM 26L and 30L:  top 3 levels 
+          ! E3SM 72L:  top 6 levels
+          !original cam formula
+          !nu_scale_top(k) = 8*(1+tanh(log(ptop_over_press))) ! active for p<4*ptop
+          nu_scale_top(k) = 16*ptop_over_press**2 / (ptop_over_press**2 + 1)
```

Comment:
Would you please replace all **2 with multiplication? I do not know how compilers would interpret those but it would be good to avoid pow issues.